resources/fileTemplates/internal/Magento Observer Class.php.ft

```diff
@@ -10,4 +10,13 @@ use Magento\Framework\Event\ObserverInterface;
 use Magento\Framework\Event\Observer;
 
 class ${NAME} implements ObserverInterface {
-}
\ No newline at end of file
+
```

Comment:
Please delete the method, this is not a part of the file template.
Moreover, see the result of the generation:
![image](https://user-images.githubusercontent.com/20116393/78670366-4bef3300-78e6-11ea-9a82-05d84bbab9b6.png)